docker/push

```diff
@@ -0,0 +1,94 @@
+#!/bin/bash
+set -e
+
+if [ -z ${IMAGE_REPO} ];
+then
+  echo "Error: IMAGE_REPO is not specified";
+  exit 1
+fi
+
+IMAGEID=$( sudo docker images -q ${IMAGE_REPO}:ci )
+echo $IMAGEID 
+if [ -z ${IMAGEID} ];
+then
+  echo "Error: unable to get IMAGEID for ${IMAGE_REPO}:ci";
+  exit 1
+fi
+
+# In case of cstor, the commit id doesn't change so using date. 
+BUILD_ID=$(git describe --tags --always)
```

Comment:
same like other PR, can we use commit id now, as container creation is moved from cstor repo to zfs repo?

Reply:
done! thanks!